test/utils.test.js

```diff
@@ -0,0 +1,59 @@
+const helper = require('../utils.js')
+
+const SECONDS_IN_DAY = 86400
+
+contract('Test Utils', async () =>  {
+
+    it("Test advanceBlock", async() => {
+        const currentBlock = await web3.eth.getBlock('latest')
+        const currentBlockNum = currentBlock.number
+
+        await helper.advanceBlock()
+        
+        const advancedBlock = await web3.eth.getBlock('latest')
+        const advanceBlockNum = advancedBlock.number
+        
+        assert.equal(currentBlockNum + 1, advanceBlockNum, "New block was not mined")
+    })
+
+    it("Test advanceTime", async() => {
+        const currentBlock = await web3.eth.getBlock('latest')
+
+        await helper.advanceTime(SECONDS_IN_DAY);
+        await helper.advanceBlock() // advanced time does not show unless block has been mined and shown in new block
+        
+        const advancedBlock = await web3.eth.getBlock('latest')
+        assert.isBelow(currentBlock.timestamp, advancedBlock.timestamp, "Time was not advanced")
+    })
+
+    it("Test advanceTimeAndBlock", async() => {
+        const currentBlock = await web3.eth.getBlock('latest')
+        await helper.advanceTimeAndBlock(SECONDS_IN_DAY)
+
+        const advancedBlock = await web3.eth.getBlock('latest')
+        assert.isBelow(currentBlock.timestamp, advancedBlock.timestamp, "Time and Block was not advanced")
+    })
+
+    it("Test takeSnapshot", async() => {
+        const snapshot = await helper.takeSnapshot()
+        const snapshotId = snapshot.result
+        assert.exists(snapshotId, "Unable to produce snapshot")
+    })
+
+    it("Test revertToSnapShot", async() => {
+        // initial data captured
```

Comment:
Not specific enough

Reply:
Here also maybe we don't need to comment? The test already describes itself